frontend/src/components/Logs/CategoryHeading/CategoryHeading.tsx

```diff
@@ -0,0 +1,17 @@
+import './CategoryHeading.styles.scss';
+
+import { Typography } from 'antd';
+import { ReactNode } from 'react';
+
+interface ICategoryHeadingProps {
+	children: ReactNode;
+}
+function CategoryHeading({ children }: ICategoryHeadingProps): JSX.Element {
+	return (
+		<Typography.Text type="secondary" className="category-heading-text">
```

Comment:
Are we considering BEM? 

https://getbem.com/introduction/

Reply:
Yes, that will be a great idea.